varscan/intervals_to_bed.cwl

```diff
@@ -1,30 +1,21 @@
 #!/usr/bin/env cwl-runner
 cwlVersion: v1.0
 class: CommandLineTool
-baseCommand: ['/usr/bin/perl', 'helper.pl']
+baseCommand: ['/bin/bash', 'helper.sh']
 requirements:
     - class: DockerRequirement
       dockerPull: "ubuntu:xenial"
     - class: InitialWorkDirRequirement
       listing:
-          - entryname: 'helper.pl'
+          - entryname: 'helper.sh'
             entry: |
-                use feature qw(say);
-
-                for my $line (<STDIN>) {
```

Comment:
It might've been nice to make the change from Perl to `grep` + `awk` in a separate commit; if we'd wanted to "fix" the Perl script, changing this `<STDIN>` to `<>` would do it.